src/features/tasks/tasksSlice.js

```diff
@@ -42,6 +44,16 @@ const initialState = {
       isLoading: false,
       errors: null,
     },
+    deleteImages: {
```

Comment:
Wasted a day or so trying to refactor this to reduce all this seeming redundancy, but it seemed a bit tricky. Doesnt make for a nice development experience

Reply:
Do you mean like trying to consolidate the `deleteImages` and `deleteImagesByFilter` state, action reducers, and thunks into one that's a bit more abstracted? If so what challenges did you run into? Obv the inputs for both are different but are there other things that were making it difficult to make less redundant?

I realize this might be easier to talk through on a huddle... I am going to be in the field most of the day tomorrow but could try to connect in the AM?

Reply:
Something we chatted about in the AM, but was mostly referring to all the actions we use to update loading state for each different type of task and is making this file less readable anytime a new task is added. Going to merge the 2 delete tasks here, but I think we may need to rethink this a bit in a separate issue